docs/src/main/paradox/rolling-updates.md

```diff
@@ -237,7 +237,7 @@ Java
 
 #### Configuration
 
-The following configuration is required, more details for each and additional configurations can be found in [reference.conf](https://github.com/akka/akka-management/blob/main/rolling-updates-kubernetes/src/main/resources/reference.conf):
+The following configuration is required, more details for each and additional configurations can be found in [reference.conf](https://github.com/akka/akka-management/blob/main/rolling-update-kubernetes/src/main/resources/reference.conf):
```

Comment:
Not sure how long the typo has been there...

I think the comment in `reference.conf` is sufficient documentation for a niche feature.